pom.xml

```diff
@@ -365,7 +365,7 @@
               <!--<arg>-agentlib:jdwp=transport=dt_socket,server=y,suspend=n,address=5005</arg> -->
             </jvmArgs>
             <!-- GWT Super Dev Mode: deploy backend correctly -->
-            <launcherDir>${deployment.parent}/${deployment.name}</launcherDir>
+            <launcherDir>optashift-employee-rostering-webapp/target/optashift-employee-rostering-webapp-${project.version}</launcherDir>
```

Comment:
This will probably prevent automatic JavaScript update in the local Wildfly instance (that is, when you run with `mvn wildfly:start wildfly:deploy`); you should also modify the ant run script so the symlink points to `optashift-employee-rostering-webapp/target/optashift-employee-rostering-webapp-${project.version}`.

Reply:
Actually, `${deployment.parent}/${deployment.name}` resolves to the full path of optashift-employee-rostering-webapp/target/optashift-employee-rostering-webapp-${project.version}, so it should be fine, but why did you need to change it?